apps/frontend/src/locales/en/translations.json

```
@@ -572,7 +572,11 @@
         "successMessage": "Your preferences have been saved.",
         "errorMessage": "Something went wrong, please try again.",
         "authErrorMessage": "Authentication failed, please try again.",
-        "unconfirmedEmailWarning": "Unconfirmed - check inbox for confirmation email"
+        "unconfirmedEmailWarning": "Unconfirmed - check inbox for confirmation email",
+        "invalidEmail": "Invalid email address",
+        "unsavedChanges": "Your changes have not been saved yet",
+        "savedChanges": "Changes saved",
+        "unsubscribed": "You were successfully unsubscribed"
```

```suggestion
        "unsubscribed": "Unsubscribed"
```

Needs to be updated as per spec - I'm fine with current copy if it's approved by John (but we'll have to update D1 PR to match if so)

---

I committed the changes, no need to overcomplicate it and we can always change it in the future.